src/Game.js

```diff
@@ -74,25 +74,42 @@ const playCard = (G, ctx, card) => {
     console.log(`${ctx.playerID} played`);
     const playerId = ctx.playerID;
     const playedCard = G.players[playerId].cards[card];
-    G.players[playerId].cards = G.players[playerId].cards.filter(c => c !== playedCard);
-    G.aboutToPlay[playerId] = null;
-
-    G.cardsPlayed.push(playedCard);
-    G.gameOver = isGameOver(G);
+    return {
+        ...G,
+        players: {
+            [playerId]: {
+                cards: G.players[playerId].cards.filter(c => c !== playedCard)
+            }
+        },
+        aboutToPlay: {
+            [playerId]: null
+        },
```

Comment:
same as above, this deletes the entire contents `aboutToPlay`...can you change it to something like this?
```suggestion
        aboutToPlay: {
            ...aboutToPlay
            [playerId]: null
        },
```

Reply:
Oh shoot, my bad! Just fixed.